assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -43,6 +43,12 @@
 		}
 	}
 
+	@media (min-width: $page-content-max-width) {
+		// Once the page content reaches it's max-width, fix the overlay notifictation to the
+		// right had side of the page content.
+		right: calc((100vw - #{$page-content-max-width}) / 2); // NOTE: this doesn't account for the WP menu collapsed state.
```

Comment:
As noted here, over the content max width the overlay notification position does not account for the collapsed state of the WordPress menu. This means that the notifications distance from the right of the page content column changes when opening and closes the menu.

We can address this in the final ticket using an approach [similar to the user input app](https://github.com/google/site-kit-wp/blob/48dfa8785cde2c5a065d31cc8970fb64466472aa/assets/sass/components/user-input/googlesitekit-user-input-module.scss#L197-L225). I will add this to the main IB to be implemented there.